modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ModalCardSubscription/index.js

```diff
@@ -0,0 +1,144 @@
+import {useQuery} from '@apollo/client';
+import ClayButton from '@clayui/button';
+import ClayIcon from '@clayui/icon';
+import ClayModal from '@clayui/modal';
+import React, {useMemo, useState} from 'react';
+import Table from '../../../../common/components/Table';
+import {getAccountSubscriptionsTerms} from '../../../../common/services/liferay/graphql/queries';
+import {status} from '../../utils/constants';
+import StatusTag from '../StatusTag';
+
+const ModalCardSubscription = ({accountSubscriptionERC, observer, onClose}) => {
+	const [activePage, setActivePage] = useState(1);
+	const MAX_ITEMS_PER_PAGE = 5;
```

Comment:
Should be good this param be default inside the `getAccountSubscriptionsTerms` query

Reply:
Fixed, changes are on commit **1f16638**, _LPS-140164 Moved items per page number to query_.

Reply:
This constant could be out of ModalCardSubscription.


Between line 9 and 11

Reply:
@kevenleone This constant was removed and it's value was put right into the query pagination params.